test/e2e-go/features/transactions/accountv2_test.go

```diff
@@ -257,7 +262,7 @@ int 1
 	a.Equal(uint64(1), value.Uint)
 
 	// 2 global state update in total, 1 local state updates
-	checkEvalDelta(t, &client, round, round+5, 2, 1)
+	checkEvalDelta(t, &client, round+2, round+5, 2, 1)
```

Comment:
The above transaction was sent on round+3, so we won't find it before that point. On the flip side, we might find the previous transaction, and run into the wrong conclusions.